pkg/helm/actions/kube_version.go

```diff
@@ -0,0 +1,30 @@
+package actions
+
+import (
+	"errors"
+	"k8s.io/client-go/discovery"
+	"k8s.io/client-go/rest"
+	"net/http"
+)
+
+func GetKubeVersion(apiEndpoint string, transport http.RoundTripper) (string, error) {
+	config := &rest.Config{
+		APIPath:   apiEndpoint,
+		Transport: transport,
+	}
+	client, err := discovery.NewDiscoveryClientForConfig(config)
+	if err != nil {
+		return "", err
+	}
+
+	kubeVersion, err := client.ServerVersion()
+	if err != nil {
+		return "", err
+	}
+
+	if kubeVersion != nil {
+		return kubeVersion.String(), errors.New("failed to get kubernetes version")
```

Comment:
hm, if kubeversion is not nill, then we have it, we should not return the error as well?

Reply:
no need to return error here, thanks
done